api/src/main/java/org/eclipse/microprofile/health/HealthGroup.java

```diff
@@ -53,5 +54,32 @@
      * @return name of the custom group
      */
     String value();
+
+    /**
+     * Support inline instantiation of the {@link HealthGroup} qualifier.
+     *
+     * @author Antoine Sabot-Durand
+     * @since 2.2
+     */
+    public static final class Literal extends AnnotationLiteral<HealthGroup> implements HealthGroup {
+
+        private static final long serialVersionUID = 1L;
```

Comment:
why do we need this?

Reply:
Only useful for passivation. I paste it from [CDI literals](https://github.com/eclipse-ee4j/cdi/blob/master/api/src/main/java/jakarta/enterprise/inject/literal/InjectLiteral.java#L32), but we can get rid of those

Reply:
If it's useful then you can keep it.